crates/pathfinder/src/storage/state.rs

```diff
@@ -2512,9 +2512,9 @@ mod tests {
             let mut connection = storage.connection().unwrap();
             let tx = connection.transaction().unwrap();
 
-            const BLOCK: Option<StarknetBlockNumber> = Some(StarknetBlockNumber(2));
+            let block = Some(StarknetBlockNumber::new_or_panic(2));
```

Comment:
This could remain `const`? although unsure if that really matters.

Reply:
I was surprised to see this being const. It's a copy type, const is just more typing for the same end result which looks out of place for reason.

Reply:
That's a fair point. I think my brain tends to `const` because of the C++ -- which is just the same as `let` without the `mut` :D